### PR TITLE
ci: fix release build on windows

### PR DESCRIPTION
**Pull request by deepak1556**
#### Description of Change

Fixes https://ci.appveyor.com/project/electron-bot/electron-x64-release/build/1.0.283

#### Checklist

- [x] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: patches/chromium/build_win_iwyu_for_smil_time.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: deepak1556 <hop2deep@gmail.com>
+Date: Wed, 30 Oct 2019 12:23:00 -0700
+Subject: build,[win]: IWYU for //third_party/blink/renderer/core/svg/animation
+
+Only affects windows release builds, identify the cause and remove this patch.
```
Comment (deepak1556) [inline patches/chromium/build_win_iwyu_for_smil_time.patch]: Tracking in https://github.com/electron/electron/issues/20784

**Comment by codebytere** [comment]
@jkleinsc would you be able to potentially run release builds on this?

**Comment by jkleinsc** [comment]
AppVeyor release builds running here:
AppVeyor release build request for electron-ia32 successful.  Check build status at https://ci.appveyor.com/project/electron-bot/electron-ia32-release/build/1.0.253
AppVeyor release build request for electron-woa successful.  Check build status at https://ci.appveyor.com/project/electron-bot/electron-woa-release/build/1.0.78
AppVeyor release build request for electron-x64 successful.  Check build status at https://ci.appveyor.com/project/electron-bot/electron-x64-release/build/1.0.284

**Comment by jkleinsc** [comment]
Release builds running here:
AppVeyor release build request for electron-woa successful.  Check build status at https://ci.appveyor.com/project/electron-bot/electron-woa-release/build/1.0.84
AppVeyor release build request for electron-x64 successful.  Check build status at https://ci.appveyor.com/project/electron-bot/electron-x64-release/build/1.0.290
AppVeyor release build request for electron-ia32 successful.  Check build status at https://ci.appveyor.com/project/electron-bot/electron-ia32-release/build/1.0.259

**Pull request merged**